10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+respectively. So our estimates were actually quite close to the population parameters: the mean was 
+about `r round(100*abs(estimates$sample_mean - population_parameters$pop_mean)/population_parameters$pop_mean, 1)`% off,
+while the standard deviation was 
+about `r round(100*abs(estimates$sample_sd - population_parameters$pop_sd)/population_parameters$pop_sd, 1)`% off.
+Note that in practice, we usually cannot compute the accuracy of the estimate, since we do not have access to the population
+parameter; if we did, we wouldn't need to estimate it!
+
+Also recall from the previous section that the point estimate can vary; if 
+we took another random sample from the population, then the value of our statistic may change.
+So then did we just get lucky with our point estimate above?
+How much does our estimate vary across different samples of size 20 in this example? Again, since we have access to the population,
+we can take many samples and plot the **sampling distribution** of the point estimates to get a sense
+for this variation. In this case, we'll use 1500 samples of size 20.
+
+```{r 11-example-means4, echo = TRUE, message = FALSE, warning = FALSE, fig.cap= "Sampling distribution of the sample means for sample size of 20"}
+samples <- rep_sample_n(airbnb, size = 20, reps = 1500)
+head(samples)
+
+sample_estimates <- samples %>% 
+    group_by(replicate) %>% 
+    summarise(sample_mean = mean(price))
+head(sample_estimates)
+
+sampling_distribution_20 <-  ggplot(sample_estimates, aes(x = sample_mean)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef") + 
+    xlab("Sample mean price per night ($)") 
+sampling_distribution_20
+```
+
+Here we see that the sampling distribution of the mean has one peak and is
+bell-shaped. Most of the estimates are between 
+about  \$`r round(quantile(sample_estimates$sample_mean)[2], -1)` and 
+\$`r round(quantile(sample_estimates$sample_mean)[4], -1)`; but there are
+a good fraction of cases outside this range (i.e., where the point estimate
+was not close to the population parameter). So it does indeed look like we were quite lucky
+when we estimated the population mean 
+with only `r round(100*abs(estimates$sample_mean - population_parameters$pop_mean)/population_parameters$pop_mean, 1)`% error.
+Let's visualize the population
+distribution, distribution of the sample, and the sampling distribution on one 
+plot to compare them.
+<!---
+```{r 11-example-means4.5}
+sample_estimates %>% 
+  summarise(mean_of_sample_means = mean(sample_mean))
+```
+Notice that the mean of the sample means is \$`r round(mean(sample_estimates$sample_mean),2)`. Recall that the population mean
+was \$`r round(mean(airbnb$price),2)`. 
+-->
+
+```{r 11-example-means5, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Comparision of population distribution, sample distribution and sampling distribution"}
+grid.arrange(population_distribution + ggtitle("Population distribution"), 
```

Comment:
maybe shrink the width of this panel plot? And should we add the sample size for the bottom plot, given that is only the sampling dist of the mean for that sample size?

Reply:
Also, you need to keep the x-axes the same here. So that the plots are comparable.

Reply:
changed the plot width, indicated sample size and changed x axis